README.md

```diff
@@ -160,6 +160,9 @@ function getAllAddons({ categoryId }: GetAllAddonsParams = {}) {
   the keyboard. If it's something you think is unfixable then use
   `$FLOW_IGNORE` instead. Please explain your rationale in the comment and link
   to a GitHub issue if possible.
+* If you're stumped on why some Flow annotations aren't working, try using
```

Comment:
Right, but when you install anything to `node_modules/.bin/` eg `node_modules/.bin/flow` `npm run flow` will just work without you needing to amend your paths/profile. This is a common paradigm and is friendly to new contributors/Windows users/folks who don't want to add a folder to their `$PATH` that could have anything in it.

I'd much rather us not rely on more globals than we need. I don't have that line in my profile, for instance and I'd rather us not need to add instructions on adding things to paths for new developers when `npm run` already works fine 😄